tools/roxvet/analyzers/testtags/analyzer.go

```diff
@@ -0,0 +1,82 @@
+package testtags
+
+import (
+	"go/ast"
+	"path/filepath"
+	"strings"
+
+	"github.com/stackrox/rox/tools/roxvet/common"
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+const doc = `Ensure that our *_test.go files include a //go:build <TAG>, otherwise they won't be run by the test target.
```

Comment:
So the reason for this failure is because the function `getPodFromFile` is within `tests/common.go` and is tagged for all our e2e tests: `//go:build test_e2e || sql_integration || compliance` 

When you then have a test file without any of these tags it can't include functions from there. I don't see this as such a likely problem for people writing new tests, because the compiler will already not autocomplete to functions from that file without having one of those build tags, and it'll complain if you try to include them without.

However one way to solve this would be for me to take tests/common.go as well as two other utility files in the tests folder and to make a package out of them, which is then included by the tests that run those functions. In that case the files in that package wouldn't need any `//go:build` directives.

Thoughts?